darwin/importer/importer.py

```diff
@@ -287,11 +313,24 @@ def import_annotations(
         # remove files missing on the server
         missing_files = [missing_file.full_path for missing_file in local_files_missing_remotely]
         parsed_files = [parsed_file for parsed_file in parsed_files if parsed_file.full_path not in missing_files]
+
         for parsed_file in track(parsed_files):
-            image_id = remote_files[parsed_file.full_path]
-            _import_annotations(
-                dataset.client, image_id, remote_classes, attributes, parsed_file.annotations, dataset, append
-            )
+            if parsed_file.annotations or (delete_for_empty and dataset.version == 2):
+                image_id = remote_files[parsed_file.full_path]
+                _import_annotations(
+                    dataset.client,
+                    image_id,
+                    remote_classes,
+                    attributes,
+                    parsed_file.annotations,
+                    dataset,
+                    append,
+                    delete_for_empty,
+                )
+            else:
+                console.print(
```

Comment:
This is gonna look really odd, we're already in `track` progress bar, so it's gonna show a empty progress bar and then this error. IMO we should look for this condition before we start the progress bar and display this warn instead.

Reply:
Ooooki. Also fixed. Progress bar is no longer destroyed. 
I also added a nice warning that lets the user know if he is using `--delete-for-empty` with a V1 dataset. Sometimes our users can be confused, so a warning should save us some swat ticket. 

Let me know !